release-notes/index.md

```diff
@@ -4,15 +4,10 @@ keywords: docker, documentation, about, technology, understanding,  release
 title: Docker release notes
 ---
 
-Find out what's new in Docker! Release notes also contain detailed information about known issues and fixes for each component. 
+Find out what's new in Docker! Release notes can be found in the product manuals:
```

Comment:
How about updating the second sentence to 'Release notes contain information about new features, improvements, known issues, and bug fixes in each release. You can find release notes for each component in the product manuals section. We suggest that you regularly visit the release notes to learn about updates.`

Reply:
Thanks! I thought to keep it light, but your suggestion looks good 🤗